Makefile.am

```diff
@@ -39,7 +39,7 @@ dist_doc_DATA = \
 	COPYING
 
 AM_CPPFLAGS = $(OPENSSL_CRYPTO_CFLAGS) -D_UNICODE
-AM_CFLAGS = -municode
+AM_CFLAGS = -municode -Werror
```

Comment:
Hi,

On Mon, May 23, 2016 at 01:45:58PM -0700, Selva Nair wrote:
> > @@ -39,7 +39,7 @@ dist_doc_DATA = \
> >  	COPYING
> >  
> >  AM_CPPFLAGS = $(OPENSSL_CRYPTO_CFLAGS) -D_UNICODE
> > -AM_CFLAGS = -municode
> > +AM_CFLAGS = -municode -Werror
> 
> We have -Wall and -Wextra turned on, adding -Werror to that could become very limiting. The code wont compile until all warnings are eliminated is not where I would like to go... Let's see what others say.

Looking at warnings and getting rid of the reasonable ones is good :-)

Elimitinating *all* warnings might just not be possible with mingw, if the
header files itself trigger warnings (which I've seen compiling on 
ubuntu 14), so -Werror on-by-default for cross-compiled code is problematic.
So I'd not go there.

gert
-- 
USENET is *not* the non-clickable part of WWW!
                                                           //www.muc.de/~gert/
Gert Doering - Munich, Germany                             gert@greenie.muc.de
fax: +49-89-35655025                        gert@net.informatik.tu-muenchen.de